content/en/continuous_integration/setup_pipelines/gitlab.md

```diff
@@ -139,6 +139,21 @@ After the integration is successfully configured, the [Pipelines][4] and [Pipeli
 
 **Note**: The Pipelines page shows data for only the default branch of each repository.
 
+
+## Enable job log collection (beta)
+
+Supported GitLab versions for job logs, only when using [object storage for job logs][6]:
+* GitLab.com (SaaS)
+* GitLab >= 14.8 (self-hosted)
+
+To enable collection of job logs the [feature flag][7] `datadog_integration_logs_collection` must be enabled in your GitLab self-hosted or GitLab.com account.
+After enabling it a new option appears in the settings of the integration: `Enable logs collection`. Check this option and save the settings.
+
+Job logs are now available in [Logs][8] and connected to the GitLab pipelines within CI Visibility.
+
+<div class="alert alert-info"><strong>Note</strong>: Logs are billed separately from CI Visibility. </br>
+Retention filters might be needed to ensure the job logs are not sampled out.</div>
```

Comment:
There are no "retention filters" in logs, but "indexes". And by default there's an index that matches everything. I would not say anything here because it confuses more than it helps.

```suggestion
<div class="alert alert-info"><strong>Note</strong>: Logs are billed separately from CI Visibility</div>
```